### PR TITLE
Resurrect moveit_planners/chomp and moveit_experimental/collision_distance_field.

### PR DESCRIPTION
**Pull request by ksatyaki**
I'm running ros-indigo. Hence, I'm not able to compile the code. Could you please check if it compiles and let me know?

### CODE REVIEW

**Comment by davetcoleman** [comment]
If you've only tested this in indigo then please re-target the PR for the indigo branch

File: moveit_experimental/README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 Experimental Packages for MoveIt!
 =================================
 
-This repository includes possibly broken/unmaintained/unfinished libraries that are being worked on or have been worked on in the past.
-They are not part of MoveIt! officially.
-No support is offered for code in this repository, but you are welcome to use it if you see fit.
```
Comment (davetcoleman) [inline moveit_experimental/README.md]: this sentence should remain - experimental is still experimental

File: moveit_planners/ompl/ompl_interface/docs/OMPLDynamicReconfigureConfig.wikidoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Autogenerated param section. Do not hand edit.
```
Comment (davetcoleman) [inline moveit_planners/ompl/ompl_interface/docs/OMPLDynamicReconfigureConfig.wikidoc]: these changes do not belong in this PR

File: moveit_planners/ompl/ompl_interface/docs/OMPLDynamicReconfigureConfig.dox [hunk 3]
```diff
@@ -0,0 +1,6 @@
+\subsubsection parameters ROS parameters
```
Comment (davetcoleman) [inline moveit_planners/ompl/ompl_interface/docs/OMPLDynamicReconfigureConfig.dox]: these changes do not belong in this PR

File: moveit_planners/ompl/ompl_interface/cfg/cpp/ompl_interface_ros/OMPLDynamicReconfigureConfig.h [hunk 4]
```diff
@@ -52,6 +52,7 @@
 
 #include <dynamic_reconfigure/config_tools.h>
 #include <moveit/macros/class_forward.h>
+
```
Comment (davetcoleman) [inline moveit_planners/ompl/ompl_interface/cfg/cpp/ompl_interface_ros/OMPLDynamicReconfigureConfig.h]: these changes do not belong in this PR

**Comment by davetcoleman** [comment]
Also, in the PR description can you give us some background on why the changes are being made and who made them. It looks like its much more than just fixing code rot / catkenizing

**Comment by ksatyaki** [comment]
I will redo the merge. It seems like there are quite a few mistakes. Sorry.
